homeassistant/components/water_heater/__init__.py

```diff
@@ -129,7 +129,7 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return await hass.data[DATA_COMPONENT].async_unload_entry(entry)
 
 
-class WaterHeaterEntityEntityDescription(EntityDescription, frozen_or_thawed=True):
+class WaterHeaterEntityDescription(EntityDescription, frozen_or_thawed=True):
```

Comment:
Although I agree with the change, and no core integrations are using this entity description, we need to check if it's used among custom integrations published on HACS before we can change this.

Reply:
Ok, let's go ahead then.